server/server.go

```diff
@@ -241,6 +243,7 @@ func (s *Server) logPid() error {
 func (s *Server) Start() {
 	s.Noticef("Starting nats-server version %s", VERSION)
 	s.Debugf("Go build version %s", s.info.GoVersion)
+	s.Debugf("GIT hash [%s]", GITHASH)
```

Comment:
Would make it a `Noticef` after the print of the version. Could be a conditional for when value is empty because not built this way.
Something like:
```
	s.Noticef("Starting nats-server version %s", VERSION)
	if GITHASH != "" {
		s.Noticef("GIT hash: %s", GITHASH)
	}
```
Would even have it be appended to the version (as in `Starting nats-server version %s%s`) with the suffix being empty string or `fmt.Sprintf(" (%s)", GITHASH)`, but afraid that some log parsing would choke on that?

Reply:
I think it should always print something but maybe it should print 'not set'.

Reply:
Agreed. Would GitCommit be ok instead?

Reply:
sure - I'll undo the docker changes, and now they retroactively work :)

Reply:
well no, they reference `version.GITCOMMIT`, not `server.GITCOMMIT`. Also, not sure we *need* the upper case. Go does not like that much.